frontend/.jshintrc

```diff
@@ -2,7 +2,8 @@
   "predef": [
     "document",
     "window",
-    "-Promise"
+    "-Promise",
+    "Ember"
```

Comment:
you shouldn't need to do this. if jshint is complaining it is because you are using a module which has not been imported.